### PR TITLE
Ignore methods not analyzed

### PR DESCRIPTION
**Pull request by saledouble**

### CODE REVIEW

**Comment by saledouble** [comment]
There is something wrong for the difference. 

The code below is not shown in the difference. It is in the branch `saledouble:new_methods_not_analyzed` but not in the base branch.

https://github.com/saledouble/Logging-Level-Evolution-Plugin/blob/04340e132f5a59e05904f0850d35aac92c40cf5c/edu.cuny.hunter.log.core/src/edu/cuny/hunter/log/core/analysis/LogAnalyzer.java#L62

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 sudo: false
 language: java
 jdk: oraclejdk8
+dist: trusty
 cache:
```
Comment (saledouble) [inline .travis.yml]: Why it is here? This line exists in both two branches。

**Comment by saledouble** [comment]
I don't need the PR. I only need to cherry pick a commit. The merged branch is too out-of-date.

File: .travis.yml (hunk 1 above)
Comment (khatchad) [reply .travis.yml]: I don't know, but the JDK is out of date, AFAIK. It needs to be `openjdk8`.

Comment (khatchad) [reply .travis.yml]: Didn't you add it?

Comment (saledouble) [reply .travis.yml]: Yeah, it is out of date. This line is another way to pass the travis building. The build failure is from the source code. I am still working on it.

**Comment by khatchad** [comment]
Please use dist: bionic and jdk: openjdk8

**Pull request closed without merging**